NREL/ssc
# Update REopt API

Fixes https://github.com/NREL/SAM/issues/1742

* Set REopt input `net_metering_limit_kw` to system capacity when net metering is enabled.
* Change REopt solver from HiGHS to SCIP to allow large numbers for tier limits (1e38).
* Specify only `max_kw` instead of both `existing_kw` and `max_kw`.

#### To test:

1. Build a) SAM-private or b) SAM open source with NREL developer key and email in private.h.

2. Create a default PV Battery / Residential and PV Battery / Commercial case and click **Get size and dispatch** on Battery Cell and System page. REopt should return non-zero battery size and dispatch array.

Note that REopt API call and optimization takes a while -- I'm not sure if the SCIP solver is slower than HiGHS. This is something we may want to investigate for the Fall 2024 release.
